cmd/examples/nse/nse.go

```diff
@@ -65,10 +71,39 @@ func (n nseConnection) RequestEndpointConnection(ctx context.Context, req *nseco
 
 func (n nseConnection) SendEndpointConnectionMechanism(ctx context.Context, req *nseconnect.EndpointConnectionMechanism) (*nseconnect.EndpointConnectionMechanismReply, error) {
 
+	interfaceName, ok := req.LocalMechanism.Parameters[NSMkeyNSELocalMechanism]
```

Comment:
Why not just call this interface_name rather than "nselocalmechanism" ?

Different types of local mechanisms will all have different sorts of parameters.  I am totally down for sharing them when it makes sense, but its probably good to name them as clearly as possible.  Or am I misunderstanding what this parameter is?